app/controllers/places_controller.rb

```diff
@@ -8,6 +8,9 @@ def index
 
   # GET /places/1 or /places/1.json
   def show
+    @place = Place.find(params[:id])
+    @storage_item = @place.storage_item
+    redirect_to new_place_storage_item_path
```

Comment:
..but if you're showing the place (with it's storage items) you don't want to redirect away again, you just want to render the "show" template (so can probably just remove this redirect?)